BGL/include/CGAL/boost/graph/parameters_interface.h

```diff
@@ -209,3 +209,12 @@ CGAL_add_named_parameter(accuracy_t, accuracy, accuracy)
 CGAL_add_named_parameter(maximum_running_time_t, maximum_running_time, maximum_running_time)
 CGAL_add_named_parameter(overlap_t, overlap, overlap)
 CGAL_add_named_parameter(maximum_normal_deviation_t, maximum_normal_deviation, maximum_normal_deviation)
+
+// shape detection
+CGAL_add_named_parameter(max_angle_t, max_angle, max_angle)
```

Comment:
if we want to be consistent with other named parameters it should be maximum, minimum, ...

Reply:
Fixed.